dev/TabView/TabView.cpp

```diff
@@ -136,18 +139,21 @@ void TabView::OnApplyTemplate()
 
     if (SharedHelpers::IsThemeShadowAvailable())
     {
```

Comment:
I haven't been following the shadow discussion super closely, why are we removing the shadows entirely pre 21h1?

Reply:
So with the new drop shadows, there's no concept of a caster/receiver. The drop shadow will render behind any element that has their own ThemeShadow (like TabViewItem). TabView's ShadowCaster happens to have its own ThemeShadow, and since it's a thin rectangular grid that runs right under the TabViewHeader area, it will cast it's own DropShadow. So with DropShadows on we'll actually end up seeing a bit of shadow poking out from the sides of the TabView content area. For the sake of simplicity, I disabled the shadow receiver/caster combo in TabView since TabViewItem will draw it's own shadow.

Reply:
But doesn't this change mean that we are removing theme shadow support for OS Versions less than 21H1? If you look at Karen's NavView PR we are keeping the theme shadow behavior when we arne't 21H1OrHigher: [https://github.com/microsoft/microsoft-ui-xaml/pull/4328/files#diff-639ba7acd5d57a0d3f04dd1195f6091c5fa12f9e66946be91ed679b840518d10R693](https://github.com/microsoft/microsoft-ui-xaml/pull/4328/files#diff-639ba7acd5d57a0d3f04dd1195f6091c5fa12f9e66946be91ed679b840518d10R693)  Which is what I was expecting to see here as well.

---
In reply to: [583924196](https://github.com/microsoft/microsoft-ui-xaml/pull/4343#discussion_r583924196) [](ancestors = 583924196)

Reply:
I'm a bit confused, do you mean I should be placing my `!SharedHelpers::Is21H1OrHigher()` check above this `SharedHelpers::IsThemeShadowAvailable()` check? I believe my current check is sufficient to say if ThemeShadow is available and we're _not_ higher than 21h1, then do the caster/receiver thing.

Reply:
Have you tried running your changes on a 20h2 machine? Do the projected shadows show up.  Looking at the code it seems like you've deleted projected shadows support for OS versions older than 20h2

Reply:
Yup, ran it on my personal 20H2 machine and the projected shadows show up. Also debugged it and made sure it went inside this block of code where it's setting receivers and giving the caster its own shadow.